### PR TITLE
Add a README how-to section on how to run a project from hCore

### PR DESCRIPTION
**Pull request by TimDiekmann**
Adds a readme section on how to run a simulation from hCore and add `--release` flags to `cargo` commands.

[Asana task](https://app.asana.com/0/1199550852792314/1201497703621221/f)

### CODE REVIEW

File: packages/engine/README.md [hunk 1]
```diff
@@ -134,6 +135,36 @@ The CLI comes with a short help page: `cli help` or `cli -h`. A more detailed ex
...
+
+In order to run a simulation from hCore, you can download it from the File menu and save by clicking on `Export Project`. Also see the [online documentation for hCloud][docs]. Save and unzip your project to a location of your choice, for example by
+
+```shell
+$ unzip my-project.zip -d my-hash-project
+```
+
+To run the simulation, [build the binaries](#project-setup--building) and pass your project to the CLI:
+
+```shell
```
Comment (Alfred-Mountfield) [inline packages/engine/README.md:150]: I think we should probably get rid of this one for now, or expand it to say to 
* run a `cargo build --release` first (which I appreciate I have suggested we remove above)
* Run the cli with the appropriate path into the target folder (which might be a bit hard to explain). We can't just say CD as relative paths are used all over the place atm. I don't think `cli` will work unless you do `./target/release/cli` but maybe I'm wrong?

File: packages/engine/README.md [hunk 2]
```diff
@@ -134,6 +135,36 @@ The CLI comes with a short help page: `cli help` or `cli -h`. A more detailed ex
...
+$ cli --project  /path/to/my-hash-project single-run --num-steps 5
+```
+
+or
+
+```shell
+$ cargo run --bin cli --release -- --project /path/to/my-hash-project single-run --num-steps 5
+```
+
+In order to see more information while the simulation is running, export `RUST_LOG` before running:
```
Comment (Alfred-Mountfield) [inline packages/engine/README.md:160]:
```suggestion
In order to see more logging information while the simulation is running, you can modify the [Rust logging level](https://docs.rs/log/latest/log/enum.Level.html) by export `RUST_LOG` before running, e.g.:
```

File: packages/engine/README.md [hunk 3]
```diff
@@ -109,7 +110,7 @@ Depending on how lightweight your OS install is, you may be missing some low lev
 * `apt-get install python3-dev` - A collection of developer utilities for Python such as header files (e.g. needed when seeing `fatal error: Python.h: No such file or directory`)
       
 ### Project Setup / Building
-* Run `cargo build`
+* Run `cargo build --all --release`
```
[the remark this answers is not in the document]
Comment (TimDiekmann) [reply packages/engine/README.md:113]: Just checked again, `--all` is not needed as `default-members` are the same as `members` in *Cargo.toml*.

File: packages/engine/README.md (hunk 1 above)
Comment (TimDiekmann) [reply packages/engine/README.md:150]: > maybe I'm wrong?

No. As currently the user have to build the binary anyway, any documentation should just user `cargo run --bin cli --`

File: packages/engine/README.md [hunk 4]
```diff
@@ -134,6 +135,33 @@ The CLI comes with a short help page: `cli help` or `cli -h`. A more detailed ex
...
+In order to download and then run a simulation from hCore, you can download it by clicking `File -> Export Project` from the toolbar on the top. For help in finding or creating, and editing, simulations in hCore, take a look at our [online documentation][docs]. Then save and unzip the downloaded project to a location of your choice, for example by
+
+```shell
+$ unzip my-project.zip -d my-hash-project
+```
+
+To run the simulation, [build the binaries](#project-setup--building) and pass your project to the CLI:
+
+```shell
+$ cargo run --bin cli -- --project /path/to/my-hash-project single-run --num-steps 5
```
Comment (Alfred-Mountfield) [inline packages/engine/README.md:153]:
```suggestion
$ cargo run --bin cli -- --project /path/to/my-hash-project single-run --num-steps <NUM-STEPS>
```

**Review by Alfred-Mountfield — APPROVED** [review]

**Comment by TimDiekmann** [comment]
Squashed commits.

**Comment by Alfred-Mountfield** [comment]
Looks good, merge away

**Pull request merged**